openfisca_us/variables/gov/states/in/tax/income/exemptions/in_aged_blind_count.py

```diff
@@ -0,0 +1,20 @@
+from openfisca_us.model_api import *
+
+
+class in_aged_blind_count(Variable):
```

Comment:
Since the IN code references the Internal Revenue Code, could you place this variable in https://github.com/PolicyEngine/openfisca-us/blob/master/openfisca_us/variables/gov/irs/income/taxable_income/deductions/standard_deduction
as `aged_blind_count` and then point both `in_aged_blind_exemptions` and `additional_standard_deduction` to it?